### DISCUSSION TITLE
Postprocessing

### DISCUSSION THREAD

**Discussion by alimostafavi24** [idaholab/moose#23765] [Q&A General] [answered]
Hello,

I wanted to export animation for my results. When I use adaptive mesh multiple .e files are created. I can see the animation with peacock, but do not know how to export it. What is the best way to export movie file when adaptive mesh is used and multiple .e files are created?

Thanks,
Ali

**Comment by GiudGiud** [comment, marked as answer]
Hello

Can you load it all in paraview? Peacock is great for getting things to work but not really for postprocessing

Guillaume

**Comment by alimostafavi24** [comment]
Yes, ParaView works well. Thanks.
